src/ssl.c

```diff
@@ -13314,16 +13314,18 @@ int wolfSSL_EVP_MD_type(const WOLFSSL_EVP_MD *md)
             WOLFSSL_MSG("no type set");
             return 0;   /* failure */
         }
-        ctx->bufUsed = 0;
-        ctx->lastUsed = 0;
-        ctx->flags   = 0;
+        if (ctx->cipherType == 0xff){
```

Comment:
Would you mind adding a define for `0xFF`, such as `#define WOLFSSL_EVP_CIPH_TYPE_INIT 0xFF` and replacing it in the places `0xFF` is used?

Reply:
It is neat. Thanks.
I defined WOLFSSL_EVP_CIPH_TYPE_INIT and rewrite 0xff to it.